app/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { RouteComponentProps } from "@reach/router";
+import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
+import TableSummary from "../TableSummary";
+
+type PageRequestReviewProps = {
+  polling?: boolean;
+} & RouteComponentProps;
+
+const PageSummary: React.FunctionComponent<PageRequestReviewProps> = ({
+  polling = true,
```

Comment:
This component is only ever rendered from the router, so may as well remove this as a prop and just set it to true in the component call below, no?

Reply:
Ah, I see you disabled it in Storybook. Good call, carry on.